### PR TITLE
10 what is the most manufactured device in history

### PR DESCRIPTION
**Pull request by sndp2202**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## What is a compiler?
 
+In computing, a compiler is a computer program that translates computer code written in one programming language (the source language) into another language (the target language).
```
Comment (Yrrol656) [inline README.md:5]: this does not belong to this pull request

**Review by emreyavuzvcc — APPROVED** [review]

**Review by Yrrol656 — COMMENTED** [review]
This is a quality answer :)
